certified-connectors/TPS & CTPS Checker/apiDefinition.swagger.json

```diff
@@ -0,0 +1,168 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "API for Querying the TPS/CTPS status of a UK telephone number",
+    "title": "TPS & CTPS Checker",
+    "version": "1.0",
+    "contact": {
+      "name": "QuartzData",
+      "url": "https://tpsmydata.co.uk/#contact",
+      "email": "info@quartzdata.co.uk"
+    }
+  },
+  "host": "api.tpsmydata.co.uk",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/",
+  "paths": {
+    "/": {
+      "get": {
+        "summary": "Query the TPS/CTPS status of UK based telephone number",
```

Comment:
Kindly remove / this character, as this is not allowed. Swagger validation is failing with the below error. 

Summary string contains any of the restricted characters [/\<>*%&?] ValuePath: /paths/~1/get/summary